spikeinterface/sortingcomponents/peak_localization.py

```diff
@@ -28,8 +27,18 @@ def localize_peaks(recording, peaks, method='center_of_mass',
         The left window before a peak in millisecond
     ms_after: float
         The left window before a peak in millisecond
-    job_kwargs: dict
-        Parameters for ChunkRecordingExecutor
+    **job_kwargs: keyword arguments for parallel processing:
+        * chunk_size or chunk_memory, or total_memory
+            - chunk_size: int
+                number of samples per chunk
+            - chunk_memory: str
+                Memory usage for each job (e.g. '100M', '1G'
+            - total_memory: str
+                Total memory usage (e.g. '500M', '2G')
+        * n_jobs: int
+            Number of jobs to use. With -1 the number of jobs is the same as number of cores
+        * progress_bar: bool
+            If True, a progress bar is printed
```

Comment:
Do not copy paster please! Because if we change one params, it will be super hard to maintain.
Make something like _shared_job kargs somewhere. and use format.